modules/backend/widgets/form/partials/_form_tabs.htm

```diff
@@ -12,10 +12,10 @@
     }
 ?>
 <div class="<?= $navCss ?>">
-    <ul class="nav nav-tabs">
+    <ul class="nav nav-tabs" <?= $tabs->linkable ? 'data-linkable' : '' ?>>
         <?php $index = 0; foreach ($tabs as $name => $fields): ?>
             <li class="<?= $index++ == 0 ? 'active' : '' ?>">
-                <a href="#<?= $type . 'tab-' . $index ?>">
+                <a href="#<?= $type . 'tab-' . $index ?>" <?= $tabs->linkable ? 'data-tab-url="#' . str_slug(trans($name)) . '"' : '' ?>>
```

Comment:
@Rike-cz I think it might actually be a better idea to not translate the `$name` before slugging it because that would create different deep links for users with different backend languages.

Reply:
ok, but something like #company_plugin_lang_tabs_tab1 doesn't look good. So do you prefer something like icons have? https://octobercms.com/docs/backend/forms#form-tab-options

Reply:
@Rike-cz I don't think it really matters what it looks like, so long as it works. I'd prefer not to add a separate config section for it because then you'd have to add each linkable tab to the config in order for that tab's link to work. A lot of modern browsers completely hide that information from the end users in the first place.

Reply:
ok, last attempt - remove lang prefix at least: `'data-tab-url="#' . preg_replace('#::lang\.(.+)$#', '\\1', $name) . '"'` :-)